default/AI/ShipDesignAI.py

```diff
@@ -633,11 +635,11 @@ def evaluate(self):
         # However, we also need to make sure, that the closer we are to requirements,
         # the better our rating is so the optimizing heuristic finds "the right way".
         if self.fuel < self.additional_specifications.minimum_fuel:
-            rating += -1000 * (self.additional_specifications.minimum_fuel - self.fuel)
+            rating += MISSING_REQUIREMENT_MULTIPLIER * (self.additional_specifications.minimum_fuel - self.fuel)
```

Comment:
The third rating calc here uses different whitespace than the first two, and I think it is the correct one.  It looks to me like we've been doing our whitespace wrong on many calculations like this.  As I read it, PEP8 explicitly says that certain specific binary operators, for equals, augmented equals (+= etc), comparisons and boolean operators (and sometimes the slice colon) should have whitespace to either side, and it strongly implies through multiple examples and statements that other binary operators should not, except that some extra whitespace can be added around a lower priority operator to help distinguish from a higher priority one.  The "c =" examples below are particularly applicable here:
> PEP8 Says:
```
Yes:
x = x*2 - 1
hypot2 = x*x + y*y
c = (a+b) * (a-b)

> No:
x = x * 2 - 1
hypot2 = x * x + y * y
c = (a + b) * (a - b)
```